art/estimators/classification/pytorch.py

```diff
@@ -1037,14 +1137,9 @@ def get_layers(self) -> List[str]:
                         import torch.nn as nn
 
                         result = []
-                        if isinstance(self._model, nn.Sequential):
```

Comment:
Does this new version also support `nn.Sequential` models?

Reply:
The granularity is maintained at module level, so i think it should.